src/dash-adapter.js

```diff
@@ -75,6 +76,18 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
   }
 
   /**
+   * Checks if dash adapter can play a given drm data.
+   * @param {Array<Object>} drmData - The drm data to check.
+   * @returns {boolean} - Whether the dash adapter can play a specific drm data.
+   * @static
+   */
+  static canPlayDrm(drmData: Array<Object>): boolean {
+    let canPlayDrm = DrmHelper.canPlayDrm(drmData);
+    DashAdapter._logger.debug('canPlayDrm result is ' + canPlayDrm.toString());
```

Comment:
Comment should be more descriptive IMHO, canPlayDrm method name is not so informative as just printing DRM (is/is not) support